opts/envdir.go

```diff
@@ -0,0 +1,43 @@
+package opts
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+/*
+Read in a directory with files acting as environment variables.
+*/
+func ParseEnvDir(directory string) ([]string, error) {
+	dir, err := ioutil.ReadDir(directory)
+	if err != nil {
+		return []string{}, err
+	}
+
+	lines := []string{}
+	for _, info := range dir {
+		// ignore directories
+		if !info.IsDir() {
+			variable := info.Name()
+			if strings.ContainsAny(variable, whiteSpaces) {
```

Comment:
I just checked out the original `envdir.c` and it does, in fact, ignore files that begin with `.`. Will update to reflect this.

For posterity, here is the actual `envdir.c`: https://gist.github.com/mattrobenolt/1e7621accfe2377dde0c